### ISSUE TITLE
[request] update RRF 3.3 -> 3.4

### ISSUE THREAD

**Issue by Moorviper** [FYSETC/FYSETC-SPIDER#131]
Hi,

is it possible that fysetc provides a rrf 3.4 image for the spider board?
That would enable the users to use can bus.

greetings
Daniel

**Comment by Moorviper** [comment]
Hi i have one of the first revision. But i guess it would be great to have a bouild for all revisions.

**Comment by jaysuk** [comment]
if its an F4 based board with 1mb flash then you can use the builds we produce for teamgloomy. If it has less flash you'll never get an updated build as its just too big to fit in 512kb. If you want to continue using RRF then best look for a different board.

**Comment by Moorviper** [comment]
yeah i had this board lying around and wanted it for extending a duet mini5+ to use it for a MMU.

**Comment by jaysuk** [comment]
You almost definitely won't be able to use it for that. I'd be surprised if
the spican code could even be squeezed into a 512kb flash MCU.
Best put it back in the drawer and find something else
